pages/index.js

```diff
@@ -47,29 +48,21 @@ export default function Home() {
                     />
                 </svg>
                 <div className="mx-auto justify-center grid sm:grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-12">
-                    <Card
-                        name="Student Name"
-                        title="Lorem Ipsum Dddddddd"
-                        projectType="Website"
-                        thumbnail="https://source.unsplash.com/random"
-                        profileIcon="https://picsum.photos/200"
-                        award="gold"
-                    />
-                    <Card
-                        name="Student Name"
-                        title="Lorem Ipsum"
-                        projectType="Discord Bot"
-                        thumbnail="https://source.unsplash.com/random"
-                        profileIcon="https://picsum.photos/200"
-                        award="silver"
-                    />
-                    <Card
-                        name="Student Name"
-                        title="Lorem Ipsum"
-                        projectType="Website"
-                        thumbnail="https://source.unsplash.com/random"
-                        profileIcon="https://picsum.photos/200"
-                    />
+                    {RepoData.map((x) => {
```

Comment:
bad practices here, destructure each prop please
```suggestion
                    {RepoData.map(({repo_name, repo_owner, project_type, repo_logo, repo_owner_icon, award}) => {
```

Reply:
resolved in 354279c